File: _정답코드/1주차/03_완전제곱수_개수_세기/solution.cpp
#include <vector>
#include <string>
#include <cmath>
using namespace std;

int solution(int n, int m) {
    int count = 0;
    for (int i = n; i <= m; ++i) {
        if ((int)sqrt(i) * (int)sqrt(i) == i) {
            count++;
        }
    }
    return count;
}

File: _정답코드/1주차/03_완전제곱수_개수_세기/solution.java
class Solution {
    public int solution(int n, int m) {
        int count = 0;
        for (int i = n; i <= m; i++) {
            int sqrt = (int)Math.sqrt(i);
            if (sqrt * sqrt == i) {
                count++;
            }
        }
        return count;
    }
}